integrationTests/testProcessorNode.go

```diff
@@ -631,11 +641,26 @@ func (tpn *TestProcessorNode) initBlockProcessor() {
 		Core:                         nil,
 		BlockChainHook:               tpn.BlockChainHookImpl,
 		ValidatorStatisticsProcessor: &mock.ValidatorStatisticsProcessorMock{},
-		EndOfEpochTrigger:            &mock.EndOfEpochTriggerStub{},
+		HeaderValidator:              headerValidator,
 		Rounder:                      &mock.RounderMock{},
 	}
 
 	if tpn.ShardCoordinator.SelfId() == sharding.MetachainShardId {
+		rounder := &mock.RounderMock{}
```

Comment:
Why not reuse the `Rounder` from `argumentsBase`?

Reply:
done